### ISSUE TITLE
Handling 'empty  pathway'

### ISSUE THREAD

**Issue by jvwong** [PathwayCommons/pathways-search#52]
Search result is returned but the view is empty.

![screenshot at apr 07 16-32-22](https://cloud.githubusercontent.com/assets/4706307/24818815/f6fb4bd6-1baf-11e7-9967-e15a309ea175.png)

How to handle this case?

Breaking case: 
P53 Signaling PathwayWIKIPATHWAYS
/#/pathway?uri=http%3A%2F%2Fidentifiers.org%2Fwikipathways%2FWP2828%2Fd36d6

SBGN
```xml
<sbgn xmlns="http://sbgn.org/libsbgn/0.2">
<map language="process description"/>
</sbgn>
```

**Comment by mj3cheun** [comment]
Appears to be a sbgn-renderer issue. I have been getting this issue today where I didn't have it yesterday as far as I remember. Also, I think collapsable nodes is broken as well. Need to ask @d2fong about it.

**Comment by mj3cheun** [comment]
Probably fixed in commit e2ae4a6b8f11eafe6496af8a42a84da770bcb8d6. Closing for now, if probably comes back will open new ticket.
